docstring.py

```diff
@@ -10,7 +10,13 @@ def get_docstring_args(fd):
     Extract docstring parameters from function definition
     """
     docstring = ast.get_docstring(fd)
-    args = re.findall(r"(\w+)\s+\:", docstring)
+    params_section = re.findall(r"(?<=Parameter)(.*)(?=Return)", docstring, re.DOTALL)
```

Comment:
It should be `Parameters` (plural) and not `Parameter`

Reply:
I used singular for both `Parameter` and `Return` just to be on the safe side. Otherwise, if by mistake, we have Parameter instead of Parameters, we cannot find the `params_section`.

Reply:
I see